src/validation/aerosol_optics/modal_size_parameters.cpp

```diff
@@ -0,0 +1,50 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <mam4xx/mam4.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <skywalker.hpp>
+#include <validation.hpp>
+
+using namespace skywalker;
+using namespace mam4;
+using namespace haero;
+using namespace modal_aer_opt;
+
+void modal_size_parameters(Ensemble *ensemble) {
+  ensemble->process([=](const Input &input, Output &output) {
+    constexpr Real zero = 0;
+
+    const Real sigma_logr_aer = input.get_array("sigma_logr_aer")[0];
+    const auto dgnumwet = input.get_array("dgnumwet");
+    const int ismethod2 = int(input.get_array("ismethod2")[0]);
+
+    std::vector<Real> radsurf(pver, zero);
+    std::vector<Real> logradsurf(pver, zero);
+
+    std::vector<std::vector<Real>> cheb_db(pver,
+                                           std::vector<Real>(ncoef, zero));
+
+    for (int kk = 0; kk < pver; ++kk) {
```

Comment:
Could this be placed inside a `Kokkos::parallel_for` block?

Reply:
It is validation test, so I do not think we need a kokkos::parallel_for.